### PR TITLE
Support of selenium 4.0-alpha

### PR DESCRIPTION
**Pull request by Bakanych**
Hi @eger-geger,
We continue to use this framework, and in the same time we're touching base of selenium 4-alpha.
It will be nice to have compatible version.

Please check this and push new version to NuGet if it looks ok.

All tests passed locally.

### CODE REVIEW

File: Selenium.HtmlElements/HtmlElements.csproj [hunk 1]
```diff
@@ -1,60 +1,69 @@
...
-    <ItemGroup Label="Nupkg">
-      <nupkg Include="**/*.nupkg" />
-    </ItemGroup>
-    <Delete Files="@(nupkg)" />
-  </Target>
-  <Target Name="AfterBuild"></Target>
-</Project>
\ No newline at end of file
+	<PropertyGroup>
+		<TargetFrameworks>net45;net5.0;netstandard2.0</TargetFrameworks>
```
Comment (eger-geger) [inline Selenium.HtmlElements/HtmlElements.csproj:3]: What is the reason to target net45 ? Is it required by WebDriver ?

Comment (eger-geger) [inline Selenium.HtmlElements/HtmlElements.csproj:3]: Isn't 'net5.0' backward compatible with 'netstandard2.0' ? See no reason to add another target unless necessarily.

**Review by eger-geger — CHANGES_REQUESTED** [review]

File: Selenium.HtmlElements/HtmlElements.csproj (hunk 1 above)
Comment (Bakanych) [reply Selenium.HtmlElements/HtmlElements.csproj:3]: yep. WebDriver 4.0 now requires 4.5+

**Comment by eger-geger** [comment]
I'll merge as is and will fix my comments with following commits.

**Pull request merged**